test/integration/rtpbroadcast.js

```diff
@@ -34,7 +35,8 @@ describe('Rtpbroadcast tests', function() {
       .then(function(session) {
         janusSession = session;
         done();
```

Comment:
Yep, forgot about that. Just had problems ones with returning promise. Will redo then.